lib/modules/platform/pr-body.ts

```diff
@@ -28,15 +35,24 @@ export function hashBody(body: string | undefined): string {
   return result;
 }
 
-function isRebaseRequested(body: string | undefined): boolean {
-  return !!body?.includes(`- [x] <!-- rebase-check -->`);
+function isRebaseRequested(body: string): boolean | undefined {
+  const match = prCheckboxRe.exec(body);
+  if (!match) {
+    return undefined;
+  }
+  return match.groups?.checkbox === '[x]';
```

Comment:
https://github.com/renovatebot/renovate/pull/17633#discussion_r964369862
```suggestion
  return prCheckboxRe.test(body);
```
comment:
>this is much easier, see other comment

reply:
>Test won't help me to know it there is no checkbox at all.
If we test for [x] we will have only binary info about the checked box. i.e. (true for [x], false for [ ] and for no checkbox at all)
but I also need this info in order to "migrate" existing onboarding PRs.
>
>This is quite defensive as we can identify this case if there is no embedded config hash comment, but i prefer checking for both.

Reply:
they will both missing on azure, so you'll migrate every time there?

Reply:
Currently we generate the onboarding PR on every run, this wont change for unsupported platforms. 
by migrating all I really mean is that for supported platforms we will generate the PR body even if the onboarding branch is up to date.

overall, migration in this context = PR update